ui/app/components/namespace-picker.ts

```diff
@@ -65,7 +65,13 @@ export default class NamespacePicker extends Component {
   }
 
   get selectedNamespace(): NamespaceOption | null {
-    return this.getSelected(this.allNamespaces, this.namespace?.path) ?? null;
+    // If a user explicitly logs in to the 'root' namespace, the path is set to 'root'.
+    // The root namespace doesn't have a set path, so when verifying the selected namespace, it returns null.
+    // Adding a check here, so if the namespace is 'root', it'll be set it to an empty string to match the root namespace.
```

Comment:
it seems like we should perhaps address this further up in the namespace service? So that the path is consistently an empty string even if a user types in `root` to the login form